### DISCUSSION TITLE
I cant get integration working in Australia?

### DISCUSSION THREAD

**Discussion by jay-oswald** [tuya/tuya-home-assistant#289] [Q&A]
I cant get the integration working in Australia, its installed, and I can "Add Integration" in HA. but it wont accept my details.

I'm thinking its because my zone is Australia, and on the Tuya IoT Platform there's no availability zone for Australia or anything close. I've selected every zone, but its not working. Not sure if there's anything I've missed.

My Username/password works properly for the v1 integration, and I'm defiantly copying and pasting the Client ID and Secret properly from the IoT platform, not sure what else I could be missing.

**Comment by tsutsuku** [comment]
Can you provide your logs?

**Reply by jay-oswald** [reply]
There dosent seam to be anything at all logged in the regular home assistant logs. I run it in docker, and usually view the logs with `docker logs homeassistant -f` and that did not show up anything. I checked the logs on the home assistant GUI at `/config/logs` and it dosent show anything either. Is there anywhere else it would be logging?

I did see this in the logs, though, but I think its from the v1 plugin? as it was in the logs before I tried to setup v2 again

```Logger: tuyaha.tuyaapi
Source: /usr/local/lib/python3.9/site-packages/tuyaha/tuyaapi.py:251
First occurred: 2:18:35 (1 occurrences)
Last logged: 2:18:35

request error, error code is HTTPSConnectionPool(host='px1.tuyaeu.com', port=443): Max retries exceeded with url: /homeassistant/skill (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa3d852b0>: Failed to establish a new connection: [Errno -3] Try again')), device None```

**Reply by tsutsuku** [reply]
https://github.com/tuya/tuya-home-assistant/wiki/How-to-Get-the-Log?

**Reply by jay-oswald** [reply]
I've done that, not getting any logs about it at all still. However im still getting Invalid Authentication or Unknown Error every time I try to set it up

**Reply by jay-oswald** [reply]
Got it working, I hadn't added my app in the IOT platform

**Reply by johnw230873** [reply]
If it helps anyone else, I kept having issues until I matched my IoT password with my tuya smart app password.

**Comment by Diramu** [comment]
I'm in Australia. It works for me.
Don't confuse the password for you tuya development account, and the password for your tuya app.
